src/plugins/foxPage/foxPage.tsx

```diff
@@ -1,14 +1,95 @@
-import { RawText } from 'components/Text'
+import { SimpleGrid, TabList, Tabs } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FoxTab } from './components/FoxTab'
+import { Layout } from './components/Layout'
+import { Total } from './components/Total'
 
 export enum FoxPageTab {
   Fox = 'Fox',
   Foxy = 'Foxy',
-  OneFox = 'OneFox',
 }
 
-export const FoxPage = (props: any) => {
-  // TODO remove console log when implementing tabs
-  // eslint-disable-next-line no-console
-  console.log(props)
-  return <RawText>TODO Fox Page</RawText>
+export enum FoxPageRoutes {
+  Fox = '/fox/fox',
+  Foxy = '/fox/foxy',
+}
+
+const FoxAssetId = 'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'
+const FoxyAssetId = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3'
+const OneFoxAssetId = 'eip155:1/erc20:0x03352d267951e96c6f7235037c5dfd2ab1466232'
+const BlueFoxIconLink = 'https://static.coincap.io/assets/icons/256/fox.png'
```

Comment:
I did not find an asset with that blue FOX icon, please suggest if you have an idea for a more clean code

Reply:
IMO we should use the regular FOX icon - this can be gotten from a selector. Asset IDs are fine to hardcode (although I'd probably expect them to be mapped from route to assetId earlier on), but icons and descriptions are things that we already have in the store

Reply:
fixed ! now the top icon is not blue anymore